suite-common/logger/src/logsSlice.ts

```diff
@@ -0,0 +1,44 @@
+import { AnyAction, createSlice, PayloadAction } from '@reduxjs/toolkit';
+
+export type LogEntry = { datetime: string; type: any; payload?: Record<any, any> };
+
+type LogsSliceState = {
+    logEntries: LogEntry[];
+};
+
+type LogsSliceRootState = {
+    logs: LogsSliceState;
+};
+
+const logsSliceInitialState: LogsSliceState = {
+    logEntries: [],
+};
+
+const MAX_ENTRIES = 200;
+
+export const logsSlice = createSlice({
+    name: 'logs',
+    initialState: logsSliceInitialState,
+    reducers: {
+        addLog: (
+            state,
+            action: PayloadAction<{
+                action: AnyAction;
+                payload: Record<string, unknown> | undefined;
+            }>,
+        ) => {
+            state.logEntries.push({
+                datetime: new Date().toUTCString(),
```

Comment:
It's not a good practice to have reducers impure even if you use createSlice, right @Nodonisko? So it's a use case for `prepare` field to get the new Date() in action creator and not inside the reducer?

Reply:
Yes, this should be done in action creator and as you mentioned there is API for that in createSlice

Reply:
https://github.com/trezor/trezor-suite/pull/7359/commits/2f69440b466e5668eede8d93f903fc5b7d445a18